src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+public class Parser {
+
+    /**
+     * splits the date and time
+     * @param description the task description
+     * @return array containing the sate and time split
+     */
+    public static String[] splitDateAndTime(String description) {
+        return description.split(" ", 2);
```

Comment:
I really liked that you moved the logic to split the arguments into the parent class. Well done!